src/libs/PolicyUtils.ts

```diff
@@ -828,6 +828,21 @@ function getCustomersOrJobsLabelNetSuite(policy: Policy | undefined, translate:
     return importedValueLabel.charAt(0).toUpperCase() + importedValueLabel.slice(1);
 }
 
+function getNetSuiteImportCustomFieldLabel(
+    policy: Policy | undefined,
+    importField: ValueOf<typeof CONST.NETSUITE_CONFIG.IMPORT_CUSTOM_FIELDS>,
+    translate: LocaleContextProps['translate'],
+): string | undefined {
+    const fieldData = policy?.connections?.netsuite?.options?.config.syncOptions?.[importField] ?? [];
+    if (fieldData.length === 0) {
+        return undefined;
+    }
+
+    const mappingSet = new Set(fieldData.map((item) => item.mapping));
+    const importedTypes = Array.from(mappingSet).map((mapping) => translate(`workspace.netsuite.import.importTypes.${mapping}.label`).toLowerCase());
```

Comment:
@yuwenmemon The GH issue mentions `Imported as tags and report fields`. Do we care for the order? I am using a loop so if the first mapping is `report` we could end up seeing `Imported as report fields and tags`. Is that okay?

Reply:
We always want tags before report fields. It should consistently say `Imported as tags and report fields` if both are being used.

Reply:
Got it. Fixed it.